websites/R/ReadTheDocs/dist/metadata.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "marzeq_",
+    "id": "500669086947344384"
+  },
+  "url": "readthedocs.io",
+  "regExp": "([a-z0-9-]+[.])*readthedocs[.](io|org)[/]?",
+  "description": {
+    "en": "Read the Docs simplifies software documentation by automating building, versioning, and hosting of your docs for you."
+  },
+  "version": "1.0.0",
+  "service": "Read the Docs",
+  "logo": "https://i.imgur.com/imTvu2G.png",
+  "thumbnail": "https://i.imgur.com/i7xxsx3.png",
+  "color": "#404A50",
+  "tags": [
+    "programming",
+    "docs",
+    "documentation",
+    "hosting"
+  ],
+  "category": "other",
+  "settings": [
+    {"title": "Main page", "value": "Viewing the main page", "id": "main"},
+    {"title": "On some documentation", "value": "Viewing %name%'s docs", "id": "viewing_docs"},
+    {"title": "On main documentation", "value": "Viewing the main docs", "id": "viewing_main_docs"},
+    {"title": "Search for", "value": "Searching for %term%", "id": "searching_for"},
+    {"title": "Search", "value": "Searching the docs", "id": "searching"},
+    {"title": "Signup", "value": "Creating an account", "id": "signup"},
+    {"title": "Login", "value": "Logging in", "id": "login"},
+    {"title": "Account settings", "value": "Managing their account", "id": "manage"},
+    {"title": "Profile", "value": "Viewing %name%'s profile", "id": "profile"},
+    {"title": "Dashboard", "value": "Viewing their dashboard", "id": "dashboard"}
```

Comment:
format it like normal
and string inputs need placeholders.

Reply:
Fixed